config/crd/bases/xgboostjob.kubeflow.org_xgboostjobs.yaml

```diff
@@ -8,6 +8,19 @@ metadata:
   creationTimestamp: null
   name: xgboostjobs.xgboostjob.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[-1:].type
+    name: State
+    type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
+  - JSONPath: .spec.ttlSecondsAfterFinished
+    name: TTL-Seconds
+    type: integer
+  - JSONPath: .spec.activeDeadlineSeconds
+    name: Active-Seconds
```

Comment:
Active-Deadline-Seconds

Reply:
KubeBuilder has a way to generate the additionalPrint Column, we can use this
https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns

Reply:
done